src/Pushup.Core/Core.Pushup.cs

```diff
@@ -24,6 +25,7 @@ public partial class Pushup : BaseUnityPlugin
         public const string Version = "1.4.0";
         internal static new ManualLogSource Logger;
 
+        private static Type MoreOutfitsType;
```

Comment:
I would add a soft BepInDependency too just to be clear that there is a non-trivial interaction.